src/sgame/sg_bot_nav.cpp

```diff
@@ -740,11 +752,17 @@ static bool BotAvoidObstacles( gentity_t *self, glm::vec3 &dir, bool ignoreGeome
 		}
 	}
 
-	if ( BotShouldJump( self, blocker, dir ) )
+	obstacle_t obst = BotDetectObstacle( self, blocker, dir );
+	if ( obst == OBSTACLE_JUMPABLE )
 	{
 		BotJump( self );
 		return false;
 	}
+	else if ( obst == OBSTACLE_LADDER )
+	{
+		dir.z += 200;
```

Comment:
do we ever have to worry about this overflowing or does this get clamped somewhere else?

Reply:
That is actually `+= 200.f`. As far as I know, the maximum z coordinate is around 65536. The maximum float value is around 3e38.